student.rb

```diff
@@ -1,10 +1,18 @@
+require_relative 'person'
+
 class Student < Person
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+  attr_accessor :classroom
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
-    @classroom = classroom
+    @classroom = nil
   end
 
   def play_hooky
     '¯(ツ)/¯'
   end
+
+  def join_classroom(classroom)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Please ensure to add the student to the classrooms' students when setting the classroom for a student